src/blockcfg/genesis_data.rs

```diff
@@ -32,12 +37,38 @@ impl fmt::Display for ParseError {
     }
 }
 
+impl std::str::FromStr for PublicKey {
+    type Err = String;
```

Comment:
It should be considered technical debt to return non-`Error` values as `Err`.

Reply:
agree